scripts/jenkins/ardana/ansible/group_vars/all/all.yml

```diff
@@ -18,6 +18,10 @@
 cloud_product: "ardana"
 #cloud_product: "crowbar"
 
+cloud_admin_user:
```

Comment:
Just a suggestion, instead of creating those kind of variables, I would create one group for the `crowbar` hosts and another for the `ardana` hosts and override those values using group vars. Also, you can put any variable exclusive to a product on its specific group vars, making sure to not load it for the other product.

Example:

```
inventory.yml:
[ardana]
ardana-admin

[crowbar]
crowbar-admin

group_vars/ardana.yml:
admin_user: "ardana"

group_vars/crowbar.yml:
admin_user: "root"
```

Reply:
@flaviodsr I agree. I will add this to the [refactoring task](https://jira.suse.de/browse/SCRD-8475) I created for this purpose.